scale-tests/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# TODO(mowsiany): the following image should match the version of scripts under
+# /tests
+FROM mesosphere/dcos-commons:latest
+ADD requirements.txt /tmp/tests-requirements.txt
+RUN pip3 install -r /tmp/tests-requirements.txt
+ENV PYTHONPATH="/spark-build/testing:/spark-build/spark-testing"
```

Comment:
Are we overwriting `PYTHONPATH` here? It's probably Ok though.

Reply:
Yes, the original instructions did that too.